tests/test_raid.py

```diff
@@ -61,7 +62,7 @@ def test_raid_error(self, run_command, fs):
         assert e.value.params[1] == 'Error!'
 
     def test_raid_missing(self, run_command, fs):
-        run_command.side_effect = subprocess.CalledProcessError(1, MDADM_DETAIL_CMD, None)
+        run_command.side_effect = ProcessException(1, MDADM_DETAIL_CMD, "stderr", "stdout")
         res = check_raid_pool(None, None)
         assert json.loads(res) == json.loads("{}")
         run_command.assert_called_once_with(MDADM_DETAIL_CMD)
```

Comment:
The test doesn't actually check the type and contents of the exception that is raised, does it? So what does it check? And do we have tests of failure cases for the updater plugin too?

Reply:
(and other plugins that use `run_command` and may fail)

Reply:
When the run_command fails in the raid context only means that thre is no raid so the plugin doesn't throw an error but just returns an empty list.
All plugin are tested for errors yes howver since the Exception raised by `run_cmd` is defined in the test we can't test the one that'd be generated within it.